igvm/vm.py

```diff
@@ -196,31 +196,37 @@ def check_serveradmin_config(self):
             if not check(value):
                 raise ConfigError(err)
 
-    def start(self, transaction=None):
+    def start(self, wait_for_ssh=True, transaction=None):
         self.hypervisor.start_vm(self)
         if not self.wait_for_running(running=True):
             raise VMError('VM did not come online in time')
 
-        host_up = wait_until(
-            str(self.dataset_obj['intern_ip']),
-            waitmsg='Waiting for SSH to respond',
-        )
-        if not host_up:
-            # If there is a network or booting error VM must be destroyed
-            # if starting has failed.
-            self.hypervisor.stop_vm_force(self)
-            raise VMError('The server is not reachable with SSH')
+        if wait_for_ssh:
```

Comment:
Checking ssh reachability is the most realiable way, even more than ping. Please read bug description again. I'm not relaxing this check for all cases. It is only for rollback.

Reply:
a rollback can also yield a broken vm because it needs to be started and starting can still be a problem. my point stays: i want igvm to fail even if rollback fails. if 60s timeout is not enough we should maybe fix this problem in a different way

Reply:
In such case I would modify it to wait for ssh but not shut down VM if it fails. Rollback should leave VM in the same state as it found it, that is running, even if failed (maybe doue to delays), if it was running before.